File: trunk/src/com/android/email/mail/CertificateValidationException.java

package com.android.email.mail;

public class CertificateValidationException extends MessagingException {
    public static final long serialVersionUID = -1;

    public CertificateValidationException(String message) {
        super(message);
    }

    public CertificateValidationException(String message, Throwable throwable) {
        super(message, throwable);
    }
}

File: mail/common/src/main/java/com/fsck/k9/mail/CertificateValidationException.kt
package com.fsck.k9.mail

import java.security.cert.X509Certificate
import net.thunderbird.core.common.exception.MessagingException

class CertificateValidationException(
    val certificateChain: List<X509Certificate>,
    cause: Throwable?,
) : MessagingException(cause)
